### PR TITLE
Link html title

### LINKED ISSUE #59: Retrieve link HTML title
When shortening an URL, Firefly should try to retrieve the remote HTML page and extract the title for display in the admin panel.

### PR DESCRIPTION
**Pull request by volgorean**
closes #59 

- Added a title field to item that defaults to null
- Added a get_title method to item that reverts to content if title is null, It isn't necessary but it removes messy logic from the templates.
- I setup Sidekiq and created a link html title worker that is pushed to queue on create.
- Added Sidekiq Web Dashboard as I asked about in Gitter.

### CODE REVIEW

**Comment by volgorean** [comment]
Also I should mention that you currently have to startup Sidekiq in another terminal window or no queue items will be processed. luckily nothing is damaged if you don't run the workers, just no html titles .
`sidekiq -e development -r ./config/environment.rb`

**Comment by volgorean** [comment]
also you need to have Redis installed and running.
default location is specified in .env
`FIREFLY_REDIS_URL=redis://localhost:6379/0`

File: spec/firefly/workers/link_html_title_worker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+require 'sidekiq/testing'
+
+Sidekiq::Testing.fake!
+# Sidekiq::Testing.inline!
```
Comment (ariejan) [inline spec/firefly/workers/link_html_title_worker_spec.rb]: If we're not using this commented out code, remove it.

**Comment by ariejan** [comment]
Looks awesome! Just one thing: coveralls. 

Coveralls is checking `config/sidekiq.rb` for test coverage. Maybe add a filter to https://github.com/ariejan/firefly/blob/master/spec/spec_helper.rb#L12 to ignore it?

**Comment by volgorean** [comment]
Cool, I added the filter. I'm not particularly familiar with Coveralls.

File: spec/firefly/workers/link_html_title_worker_spec.rb (hunk 1 above)
Comment (volgorean) [inline spec/firefly/workers/link_html_title_worker_spec.rb]: Oops, ok it's been removed.

**Pull request merged**